### PR TITLE
Fixes SaturnX

### LINKED ISSUE #66377: SaturnX doesn't turn you invisible anymore, and ONLY turns your eyeballs invisible
Reporting client version: 514.1568

## Round ID:
[181927](https://scrubby.melonmesa.com/round/181927)

## Reproduction:

- Drink SaturnX. 
- Stay completely visible, defeating the entire point of the chemical
- Notice that your eyes are now transparent, and ONLY your eyes
- image

Likely caused by KapuLimbs.

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Fixes SaturnX not making you invisible, because the new code needed for it to work was removed during Kapu's limb refactor (I assume because the downstream they were from didn't have SaturnX), but it did need some work to work with the new system.
This time there's less snowflake checks (like just removing greyscale/gender/some other thing I already forgot because I'm suppressing it from my memory, from the user's body.

Video of it in-game (ignore the lag spike near the end, I had a breakpoint):
https://user-images.githubusercontent.com/53777086/188973251-26610e50-964a-4ce0-97b7-c86159a947d6.mov

## Why It's Good For The Game

SaturnX works again, awesome!
Closes https://github.com/tgstation/tgstation/issues/66377
Closes https://tgstation13.org/phpBB/viewtopic.php?f=5&t=32421

## Changelog

:cl:
fix: SaturnX now properly turns you invisible again.
/:cl:

### CODE REVIEW

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 1]
```diff
@@ -26,6 +26,7 @@
 	var/change_exempt_flags
 
 	var/is_husked = FALSE
+	var/is_invisible = FALSE
```
Comment (san7890) [inline code/modules/surgery/bodyparts/_bodyparts.dm:29]: Erm... docs? 🤓

Comment (JohnFulpWillard) [reply code/modules/surgery/bodyparts/_bodyparts.dm:29]: nah

**Review by Kapu1178 — CHANGES_REQUESTED** [review]
the invisible var needs to be added to the key generator, bodypart/proc/generate_icon_key. or else it wont render without a force update.

**Review by Kapu1178 — APPROVED** [review]
dont review code on mobile kids

**Comment by JohnFulpWillard** [comment]
Moths, Plasmamen, and Monkeys all have unique icons that don't have invisibility limbs (which do have unique animations), which means this fix isn't fully complete yet.

Unless a spriter is willing to add species-unique invisibility limbs, I think I'm gonna have to make them all share the same human icon.

**Comment by Mothblocks** [comment]
it would be awesome if this had screenshot tests

**Comment by weesnaw-ops** [comment]
> it would be awesome if this had screenshot tests

800px-HD_transparent_picture

**Comment by JohnFulpWillard** [comment]
> it would be awesome if this had screenshot tests

Added one

**Comment by Mothblocks** [comment]
erm, they don't look very invisible to me

**Comment by JohnFulpWillard** [comment]
That's the first frame of the animation, they start blue and fade out.

**Comment by Mothblocks** [comment]
oh

**Comment by Mothblocks** [comment]
might be good idea to remove `no_anim = TRUE` then?

**Comment by JohnFulpWillard** [comment]
> might be good idea to remove `no_anim = TRUE` then?

yes good idea, done now

**Review by Mothblocks — APPROVED** [review]

**Pull request merged**